test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -116,6 +116,52 @@ Describe "Start-Process" -Tags @("Feature") {
    }
 }
 
+Describe "Start-Process -Timeout" -Tags "Feature","Slow" {
+
+    BeforeAll {
+        if ($IsWindows) {
+            $pingParam = "-n 30 localhost"
+        }
+        elseif ($IsLinux -Or $IsMacOS) {
+            $pingParam = "-c 30 localhost"
+        }
+
+        # Find where test/powershell is so we can find the testexe command relative to it
+        $powershellTestDir = $PSScriptRoot
```

Comment:
You shouldn't have to search for testexe, I think the current pattern is to expect using `Start-PSPester` which builds testexe and puts it into the path

Reply:
Sorry, didn't realise it worked that way. Shall change to just use `textexe`.